src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+function Navbar() {
+  return (
+    <div className="nav-bar">
+      <div className="nav-con">
+        <nav className="nav-link">
+          <h1><a className="logo" href="/">Bookstore CMS</a></h1>
+          <li><a href="/">Books</a></li>
+          <li><a href="/">Catagories</a></li>
```

Comment:
- [ ] Good job working on "Bookstore: initialize project with components" 😃, however, currently your `Categories`  path is not working. Your `nav` should contain 2 `<Link>`s. One to `/ `and one to `/categories`. 😃. So inside of this `<li><a href="/">Catagories</a></li>`  your  `href` suppose have something like this `<li><a href="/Categories">Catagories</a></li>`. Kindly fix it 💪.